pageserver/Cargo.toml

```diff
@@ -35,6 +35,7 @@ scopeguard = "1.1.0"
 rust-s3 = { version = "0.27.0-rc4", features = ["no-verify-ssl"] }
 async-trait = "0.1"
 const_format = "0.2.21"
+tracing = "0.1.27"
```

Comment:
That did not get removed though? 

All `log::debug!` calls should be done as `tracing::debug!` with the new approach, and tracing has its own counterparts for log levels and other structs to replace the log ones, so we should not rely on the crate in our project still?

That "missing log" case discussed separately also benefits from using `tracing` instead of `log` instead 😄